src/Super_Admin_Command.php

```diff
@@ -154,18 +155,26 @@ public function remove( $args, $_ ) {
 			WP_CLI::error( 'No super admins to revoke super-admin privileges from.' );
 		}
 
-		$users = $this->fetcher->get_many( $args );
-		$user_logins = $users ? array_values( array_unique( wp_list_pluck( $users, 'user_login' ) ) ) : array();
+		$users             = $this->fetcher->get_many( $args );
+		$user_logins       = $users ? array_values( array_unique( wp_list_pluck( $users, 'user_login' ) ) ) : array();
 		$user_logins_count = count( $user_logins );
 
 		if ( $user_logins_count < count( $args ) ) {
 			$flipped_user_logins = array_flip( $user_logins );
 			// Fetcher has already warned so don't bother here, but continue with any args that are possible login names to cater for invalid users in the site options meta.
 
-			$user_logins = array_merge( $user_logins, array_unique( array_filter( $args, function ( $v ) use ( $flipped_user_logins ) {
-				// Exclude numeric and email-like logins (login names can be email-like but ignore this given the circumstances).
-				return ! isset( $flipped_user_logins[ $v ] ) && ! is_numeric( $v ) && ! is_email( $v );
-			} ) ) );
+			$user_logins       = array_merge(
```

Comment:
For a future iteration, I'd like to suggest splitting these nested function calls to separate lines to improve readability.
```php
			$filtered_args     = array_filter(
				$args,
				function ( $v ) use ( $flipped_user_logins ) {
					// Exclude numeric and email-like logins (login names can be email-like but ignore this given the circumstances).
					return ! isset( $flipped_user_logins[ $v ] ) && ! is_numeric( $v ) && ! is_email( $v );
				}
			);
			$filtered_args     = array_unique( $filtered_args );
			$user_logins       = array_merge( $user_logins, $filtered_args );
```